pkgs/development/libraries/libvirt/default.nix

```diff
@@ -43,7 +43,9 @@ stdenv.mkDerivation rec {
     # do not use "''${qemu_kvm}/bin/qemu-kvm" to avoid bound VMs to particular qemu derivations
     substituteInPlace src/lxc/lxc_conf.c \
       --replace 'lxc_path,' '"/run/libvirt/nix-emulators/libvirt_lxc",'
-  '' + ''
+  ''
+  # "patchShebangs" causes issues in the Darwin build
```

Comment:
What issues? Either explain here or refer to where they are explained (maybe this PR).

Reply:
good point, I'll add details into the description and point to the PR. Or if anyone can spot the cause, we could also apply a proper fix ;)